dealership.rb

```diff
@@ -1,47 +1,29 @@
-class Car
-  # I need to encapsulate these objects inside the dealership...
-end
-
 class Dealership
+
+  attr_reader :cars
+
   def initialize(cars = nil)
     @cars = cars || []
   end
 
   def find_make(make)
-    cars_by_make = []
-    @cars.each do |car|
-      cars_by_make << car if car.make == make
-    end
-    cars_by_make
+    cars.select{|obj| obj.make == make}
```

Comment:
Don't forget to standardize inputs. What id the user is looking for a BMW by typing bMw. 

Always assume your users are dumb.